web/src/render/useNotificationsRenderer.ts

```diff
@@ -105,7 +106,9 @@ export const useNotificationsRenderer: UsePixiRenderer = ({
       refreshInterval: REFRESH_INTERVAL,
       isPaused: () => !initialized,
       onSuccess: (newNotifications) => {
-        if (newNotifications.length > 0) {
+        const { length } = newNotifications;
+        trackAnalyticsEvent("new-notifications", { length });
```

Comment:
this is going to fire a huge number of times - and I don't see any value in tracking it. what do you want to measure?

Reply:
We want to be sure the user gets notifications generated. Do you have any suggestions on how to do it better?

Reply:
it's now prevent from logging when there are no notifications